.github/pull_request_template.md

```diff
@@ -1,3 +1,7 @@
+---
+labels: kind/skip-release-notes
```

Comment:
Are all the PRs now labeled with `skip-release-notes`?

Reply:
will test this soon and let you know